sim_consistency/cli.py

```diff
@@ -76,7 +76,8 @@ def main_model_sim(base):
                                                             unbiased=base.unbiased,
                                                             device=base.device,
                                                             sigma=base.sigma,
-                                                            max_workers=base.max_workers)
+                                                            max_workers=base.max_workers,
+                                                            gromov_parameter=base.gromov_parameter)
     # Save the similarity matrix
     out_path = os.path.join(base.output_root, dataset_name, method_slug)
     if not os.path.exists(out_path):
```

Comment:
Ok, so maybe we have to do the file handling above because `comput_sim_matrix` can store data, which we want to be stored at the right position.

Reply:
I think storing all Transport maps in Memory might scale badly, as each is ~800MB. I now pass the output_root+dataset_name as an argument, while the method_slug is called internally.